#getting information of a stored object
#objectID		
uint32 objectID
---
#object name
string objectName
#object pose
geometry_msgs/Pose objectPose
#object classID
uint32 classID
#Reference ID in household object DB
uint32 HHobjectID
#Time stamp for the object
string lastseenTime